crates/integration_test/src/tests/lifecycle/exec.rs

```diff
@@ -0,0 +1,20 @@
+use super::get_result_from_output;
+use crate::utils::get_runtime_path;
+use std::path::Path;
+use std::process::{Command, Stdio};
+use test_framework::TestResult;
+
+pub fn exec(project_path: &Path, id: &str, exec_cmd: &str) -> TestResult {
+    let res = Command::new(get_runtime_path())
+        .stdout(Stdio::piped())
+        .stderr(Stdio::piped())
+        .arg("--root")
+        .arg(project_path.join("runtime"))
+        .arg("exec")
+        .arg(id)
+        .arg(exec_cmd)
+        .spawn()
+        .expect("failed to execute exec command")
+        .wait_with_output();
+    get_result_from_output(res)
```

Comment:
In case we change the command to `echo "Hello world"` , you will have to manually get the stdout and stderr here, and check for no-error and that stdout has the expected output.

Reply:
👍